src/services/item/controller.ts

```diff
@@ -27,6 +27,9 @@ import { Ordered } from './interfaces/requests';
 import { ItemSearchParams } from './types';
 import { ItemOpFeedbackEvent, memberItemsTopic } from './ws/events';
 
+export interface ItemName {
+  [key: string]: string;
+}
```

Comment:
I generally prefer type aliases for custom defined types.
Maybe we could change the type naming to express better that is is related to an association between the item Id and the a new name. Below is a suggestion:
```suggestion
export type ItemIdToNameMap = {
  [key: string]: string;
}
```
The type name is not final, please feel free to choose a better one 😄